code/game/objects/machinery/computer/skills.dm

```diff
@@ -21,6 +21,12 @@
 	var/sortBy = "name"
 	var/order = 1 // -1 = Descending - 1 = Ascending
 
+/proc/I_runtime_on_rails()
+	var/list/L = list()
+	var/chumpo = "fetusin"
+	var/chumpo2 = "fetus"
+	L[chumpo[chumpo2]] = "???"
+	return L
```

Comment:
What the actual fuck is this

Reply:
Some garbage lines I almost subconsciously wrote down as I tried to figure out an issue I later resolved but then forgot to remove. kek.